ceph-ansible-nightly/build/build

```diff
@@ -25,6 +25,11 @@ function find_latest_tag {
   page=1
   while response=$(curl -s "https://quay.ceph.io/api/v1/repository/ceph-ci/daemon/tag/?page=${page}&limit=${page_size}"); do
     tag=$(echo "${response}" | jq -r ".tags[] | select((.name | contains(\"stable\")) and (.name | contains(\"${release}-centos-${el}-x86_64\"))) | .name")
+    let code=$?
```

Comment:
little concerned about mixing 'let' (explicitly for arithmetic statements) and '[ !=' (which I'm not sure is); reading bash docs

Reply:
I think I _meant_ `local=`, but wrote `let=` instead. According to the docs it's fine, but I'm going to change it anyway.